packages/editor-ui/src/components/CredentialEdit/docs.ts

```diff
@@ -0,0 +1,7 @@
+export const CREDENTIAL_MARKDOWN_DOCS: Record<string, string> = {
+	aws: 'Configure this credential:\n\n- Select your AWS **Region**.\n- Log in to your <a href="https://aws.amazon.com/" target="_blank">AWS</a> account.\n- Generate your access key pair:\n  - Open the AWS [IAM console](https://console.aws.amazon.com/iam) and open your user menu.\n  - Select **Security credentials**.\n  - Create a new access key pair in the **Access Keys** section.\n  - Reveal the **Access Key ID** and **Secret Access Key** and enter them in n8n.\n- To use a **temporary security credential**, turn this option on and add a **Session token**.\n- If you use Amazon Virtual Private Cloud <a href="https://aws.amazon.com/vpc/" target="_blank">VPC</a> to host n8n, you can establish a connection between your VPC and some apps. Use **Custom Endpoints** to enter relevant custom endpoint(s) for this connection.\n\nOpen the full Docs for more detailed instructions.',
```

Comment:
Since these are just strings, they could also be moved to the translations file.